sonic_sfp/sfputilbase.py

```diff
@@ -923,6 +946,10 @@ def get_transceiver_dom_info_dict(self, port_num):
             if sfpd_obj is None:
                 return None
 
+            sfpdth_obj = sff8436DomThreshold()
```

Comment:
where is this sff8436DomThreshold() defined? shouldn't it been removed?

Reply:
Hi @keboliu 
Removed the irrelevant code